### PR TITLE
dialects: (llvm) add constant, return, call intrinsic

### PR DESCRIPTION
**Pull request by superlopuh**

### CODE REVIEW

**Review by AntonLydike — APPROVED** [review]
Thanks for the renaming of the LLVM operations! That was needed!

File: xdsl/dialects/llvm.py [hunk 1]
```diff
@@ -673,31 +677,74 @@ def get(ptr_type: LLVMPointerType | None = None):
...
+class ExtractValueOp(IRDLOperation):
+    """
+    https://mlir.llvm.org/docs/Dialects/LLVM/#llvmextractvalue-mlirllvmextractvalueop
+    """
+
     name = "llvm.extractvalue"
 
     position: DenseArrayBase = attr_def(DenseArrayBase)
-    container: Operand = operand_def(AnyAttr())
+    container: Operand = operand_def(Attribute)
```
Comment (webmiche) [inline xdsl/dialects/llvm.py:688]:
```suggestion
    container: Operand = operand_def()
```

I am getting confused again with our syntax, shouldn't this work as well?

**Review by webmiche — APPROVED** [review]

File: xdsl/dialects/llvm.py (hunk 1 above)
Comment (superlopuh) [reply xdsl/dialects/llvm.py:688]: All three are equivalent, you're right, as well as `container = operand_def()`

Comment (superlopuh) [reply xdsl/dialects/llvm.py:688]: I don't really have a strong preference for either quite yet, maybe one will form over time

**Pull request merged**